### DISCUSSION TITLE
Alphabetic Set Seed Search

### DISCUSSION THREAD

**Discussion by magnitood** [Zodsmar/SeedSearcherStandaloneTool#84] [Ideas]
Is it possible to add Alphabetic seed search using a list of set seeds.

**Comment by Zodsmar** [comment]
What do you mean?

**Reply by magnitood** [reply]
Like when you enter a word as a seed in the seed bar, it generates that specific world. So is it possible to search from a list of words in a text file? Also love your work bro. Really appreciate it

**Reply by Zodsmar** [reply]
the words convert to seeded numbers.

**Reply by magnitood** [reply]
Yup exactly

**Reply by Zodsmar** [reply]
I can look into adding that as a system

**Reply by magnitood** [reply]
Thanks man, really appreciate

**Reply by Zodsmar** [reply]
Added this in my most recent commit. Will be in the next version.
7141aec
